crates/uv/src/commands/project/tree.rs

```diff
@@ -284,6 +285,9 @@ pub(crate) async fn tree(
         no_dedupe,
         invert,
     );
+    if show_sizes {
+        tree = tree.with_show_sizes(true);
+    }
```

Comment:
Why use a `.with...` pattern here instead of passing the value to `new` like all the other settings?